src/modules/core/components/Fields/DatePicker/TimePicker.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useEffect, useMemo, useRef, useState } from 'react';
+import { format } from 'date-fns';
+import { defineMessages } from 'react-intl';
+import { flip, useFloating } from '@floating-ui/react-dom';
+import classnames from 'classnames';
+
+import Icon from '~core/Icon';
+import styles from './DatePicker.css';
+import { DEFAULT_TIME_FORMAT } from './constants';
```

Comment:
I'm always the one who forgets to check this, but styles come last in the imports list ⬆️🆙